### PR TITLE
ceph-common: ceph-base is not part of rhstorage, remove it

### PR DESCRIPTION
**Pull request by alfredodeza**
Solves regression when fixing https://bugzilla.redhat.com/show_bug.cgi?id=1339096

### CODE REVIEW

**Comment by leseb** [comment]
Should we try to install something else instead? If we don't clients won't have any packages installed.
I believe the package does exist on Jewel right?
So the issue impacts everything before RHCS 2.0.

Am i right?

**Comment by alfredodeza** [comment]
@leseb we never did install it in the first place (before the "rh storage on redhat" install task) and we haven't installed that for debian either.

**Comment by leseb** [comment]
Yes but we only install it when the the client role is enabled. So this will only fail if the client role is enabled.
I looked at the bz, this regression is weird, this shouldn't be happening.

**Comment by alfredodeza** [comment]
@leseb I guess this is happening because there is no `client_group_name` defined in downstream. Maybe the fix is to define that so this skips properly? 

But even then, we would implicitly allow the definition of `client_group_name` which would cause the error of `ceph-base` not being available

**Comment by alfredodeza** [comment]
@leseb what about both: define `client_group_name` but also remove this from the tasks, there is no point in making ansible skip something that is not even provided

**Comment by leseb** [comment]
Yes we should check and see if we need to update `ceph-common` vars to add `client_group_name`
I know but aren't you sure this is "just" an issue with 1.3 and that ceph-base is available in 2.0?
If the package does not exist in the end, then yes let's remove it but then we might need a replacement for it.
We have been having a lot of requests for a client role, so I'd like to keep it working :)

**Comment by alfredodeza** [comment]
@leseb you are right, `ceph-base` does exist. But there is no need to install it explicitly as it is being pulled by ceph-osd, ceph-mds, and ceph-mon

**Comment by ktdreyer** [comment]
> We have been having a lot of requests for a client role, so I'd like to keep it working :)

"ceph-base" is the shared set of files among the various servers (mon/osd/mds), and does not relate to the client (that package is "ceph-common")

( confusing names :fireworks: )

File: roles/ceph-common/tasks/installs/install_rh_storage_on_redhat.yml [hunk 1]
```diff
@@ -45,9 +45,9 @@
   when:
     - mds_group_name in group_names
 
-- name: install red hat storage ceph base
+- name: install red hat storage ceph-common
   yum:
-    name: "ceph-base"
+    name: "ceph-common"
```
Comment (leseb) [inline roles/ceph-common/tasks/installs/install_rh_storage_on_redhat.yml]: will this package be available in Jewel and later?

Comment (ktdreyer) [inline roles/ceph-common/tasks/installs/install_rh_storage_on_redhat.yml]: Yes, ceph-common's been around for a long time (pre-firefly).

Comment (leseb) [inline roles/ceph-common/tasks/installs/install_rh_storage_on_redhat.yml]: Cool thanks Ken

**Comment by leseb** [comment]
@alfredodeza thanks for updating this, if you guys are good with this, so am I :)

**Pull request merged**